src/lib/types/HAPServiceNodeType.ts

```diff
@@ -23,6 +24,8 @@ type HAPServiceNodeType = Node & {
     onCharacteristicGet: any
     onCharacteristicSet: any
     onCharacteristicChange: any
+    _flow: FlowType
+    _alias: string
```

Comment:
Could we create a new internal Node type and add those new fields there?
Then this (new) Node type could be also used for HAPHostNodeType (for future use)

Reply:
I extracted it into a type I called `NodeType` which I would assume follows your naming conventions. Are you happy with that? Otherwise I could go with `ExtendedNodeType` or just `Node` or `NRCHKBNode[Type]`.